docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -164,77 +216,141 @@ The [Db2 driver](https://repo1.maven.org/maven2/com/ibm/db2/jcc/11.5.0.0/jcc-11.
 
 1. Run in your favorite terminal, and you should see no errors in the output. You should see output like:
 
-```
-[jcc][10516][13709]Test Connection Successful.
-```
+    ```
+    [jcc][10516][13709]Test Connection Successful.
+    ```
+
+#### Test Db2 Auth Access
+1. Copy the command to your code editor
+
+    ``` bash
+    java -cp /path/to/db2jcc4.jar com.ibm.db2.jcc.DB2Jcc  -url "jdbc:db2://<DB2-HOSTNAME>:50001/bludb:user=<userid>;password=<your_password>;sslConnection=true;sslTrustStoreLocation=/path/to/truststore.jks;sslTrustStorePassword=<TRUSTSTORE-PASSWORD>;"
+    ```
+
+    - Note: Don't forget the trailing `;` in the URL. Some of the documented examples don't include it, but it is required in order for the connection to work, although this may be fixed in a future driver release. This only affects this test URL, not the actual FHIR server configuration.
+
+1. Replace the following values with your service details:
+    - `/path/to/db2jcc4.jar` : replace with the path to your driver jar.
+    - `<DB-HOSTNAME>`: the hostname of your Db2 service from the Service Credentials page
+    - `<userid>`: The userid to acecss the db
+    - `<your_password>`: The password to access the db with
+    - `<TRUSTSTORE-PASSWORD>`: the password for your truststore
+
+1. Run in your favorite terminal, and you should see no errors in the output. You should see output like:
+
+    ``` bash
+    [jcc][10516][13709]Test Connection Successful.
+    ```
 
 ### **Deploy the IBM FHIR Server schema**
 
 Now that you've created the database and credentials, use the `fhir-persistence-schema` utility to deploy the IBM FHIR Server schema:
 
 1. download the `fhir-persistence-schema` cli jar from the corresponding project release: https://github.com/IBM/FHIR/releases
 
-2. create a properties file named db2.properties with the db2 connection info from IBM Cloud; for example:
-    ```
-    db.host=dashdb-txn-***-***********.services.dal.bluemix.net
+2. create a properties file named db2.properties with the Db2 Admin connection info from IBM Cloud; for example:
+
+    ``` sh
+    db.host=1-2-3-4.databases.appdomain.cloud
     db.port=50001
-    db.database=BLUDB
-    user=bluadmin
-    password=bluadmin-password-very-secret
+    db.database=bluedb
```

Comment:
did they change the service?  in the past it was ALWAYS bludb (no `e`)

Reply:
great catch, fixed